operate/services/manage.py

```diff
@@ -1332,6 +1348,80 @@ def fund_service(  # pylint: disable=too-many-arguments,too-many-locals
                 chain_type=ledger_config.chain,
             )
 
+    def drain_service(
+        self,
+        hash: str,
+        withdrawal_address: str,
+    ) -> None:
+        """Drain the funds out of service safe and wallet."""
+        self.logger.info(f"Draining service: {hash}")
+        service = self.load_or_create(hash=hash)
+        chain_config = service.chain_configs[service.home_chain_id]
+        ledger_config = chain_config.ledger_config
+        chain_data = chain_config.chain_data
+        wallet_manager = self.wallet_manager.load(ledger_config.type)
+        ledger_api = wallet_manager.ledger_api(
+            chain_type=ledger_config.chain, rpc=ledger_config.rpc
+        )
+        ethereum_crypto = EthereumCrypto(
+            private_key_path=service.path / "deployment" / "ethereum_private_key.txt",
+        )
+
+        # drain OLAS and wxDAI from service safe
```

Comment:
Possible more tokens need to be considered, e.g., USDC for Optimus

Reply:
then, we much specify these tokens somewhere right? Perhaps in the service template?